types/enzyme/index.d.ts

```diff
@@ -300,7 +307,10 @@ export interface CommonWrapper<P = {}, S = {}, C = Component<P, S>> {
      * Returns an html-like string of the wrapper for debugging purposes. Useful to print out to the console when
      * tests are not passing when you expect them to.
      */
-    debug(): string;
+    debug(options?: {
+        ignoreProps?: boolean; // Whether props should be omitted in the resulting string. Props are included by default.
```

Comment:
These would be useful as JSDoc comments so they show up in intellisense. Something like this, I think:
```suggestion
        /** Whether props should be omitted in the resulting string. Props are included by default. */
        ignoreProps?: boolean; 
```

Reply:
@jwbay great suggestion! didn't know that intellisense picked definitions up that way. thanks :)